pyan/analyzer.py

```diff
@@ -378,6 +378,16 @@ def visit_ImportFrom(self, node):
         #
         # https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time
         from_node = self.get_node_of_current_namespace()
+        # resolve relative imports 'None' such as "from . import foo"
+        if node.module is None:
+            self.logger.debug("Old ImportFrom: from %s import %s, %s:%s" % (node.module, [format_alias(x) for x in node.names], self.filename, node.lineno))
```

Comment:
This debug message could be made clearer.

Here `node.module is None`, so the message will always display "from None". Perhaps something like `'.' * node.level` would be more useful?

Also, perhaps "ImportFrom (original):" could be clearer instead of "Old ImportFrom:". "Old" could be interpreted as "obsolete", whereas "original" unambiguously means "what the AST had before any processing".